alg/teca_temporal_reduction.py

```diff
@@ -254,24 +254,96 @@ def update(self, out_array, in_array):
                         if out_array.itemsize < 8 else \
                         out_array.astype(float64)
```

Comment:
is this intended to be `in_array.astype(float64)`?

Reply:
I think the use of out_array is correct here. I can see why you are asking. the diff is confusing. Here is the code.
```python
                # don't use integer types for this calculation
                if in_array.dtype.kind == 'i':
                    in_array = in_array.astype(np.float32) \
                        if in_array.itemsize < 8 else \
                        in_array.astype(float64)
                                                                                                                                                                                             
                if out_array.dtype.kind == 'i':
                    out_array = out_array.astype(np.float32) \
                        if out_array.itemsize < 8 else \
                        out_array.astype(float64)

```